src/wordlift/external-plugin-hooks/avada-builder/class-avada-builder-support.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Wordlift\External_Plugin_Hooks\Avada_Builder;
+
+/**
+ * Avada Builder Support.
+ *
+ * @since 3.39.2
+ * @author Mahbub Hasan Imon <mahbub@wordlift.io>
+ */
+class Avada_Builder_Support {
+
+	public function __construct() {
+		add_filter( 'wl_post_excerpt_post_content', array( $this, 'post_excerpt_post_content' ), 10, 2 );
```

Comment:
does it make sense to move the no editor analysis filter hook to here since you are checking if avada is active ?